.ddev/commands/web/clone

```diff
@@ -1,21 +1,52 @@
 #!/bin/bash
 
 ## Description: Checkout a module using git clone
-## Usage: clone MODULE_NAME [-y]
-## Example: "ddev clone admin_toolbar" or "ddev clone admin_toolbar -y"
+## Usage: clone [-b BRANCH_NAME | -t TAG_NAME] [-y] MODULE_NAME
+## Example: "ddev clone admin_toolbar" or "ddev clone -b develop -y admin_toolbar"
 
-if [ $# -eq 0 ]
-then
-  echo "You must specify a module; e.g. 'ddev check admin_toolbar'"
+help() {
+  echo "Usage: clone [-b BRANCH_NAME | -t TAG_NAME] [-y] MODULE_NAME"
+  echo "Options:"
+  echo "  -b    Specify a branch to check out."
+  echo "  -t    Specify a tag to check out."
+  echo "  -y    Clone the repo as a maintainer."
   exit
-fi
+}
 
 TYPE="contributor"
-MODULE="$1"
 
-if [ $# -eq 2 ] && [ "$2" = "-y" ]
+while getopts 'b:t:m' opt
```

Comment:
Should this be `-m` or `-y`? Or should we just default to ssh checkouts and make https the variant?

Reply:
That should be `y`. However, I like the idea of defaulting to ssh checkouts because we can make `-c` the option. I went back on using `-m` because I feared it would become confusing with `MODULE_NAME`.